docs/rfcs/XXXX_storage_backends.md

```diff
@@ -0,0 +1,248 @@
+> A simple representation of how Data Tracker and its resources are currently allocated.
+
+The Data Tracker relies on a data storage system to both save files that it receives and to account for other files that have been deposited externally. The current method that is used is adequate for a single tenant operating in a single AWS account, but needs to be re-thought for multiple tenants that  have more sophisticated storage needs.
+
+### Study Buckets
+
+Studies are central entities in the Data Tracker. They partition data that is collected and submitted by different research projects and investigators. For this reason, it was determined that each study would have its own bucket to segment it from other collections of data.
+
+#### Creation
+The Data Tracker automatically creates and configures buckets upon creation of a new study. These buckets are created in the same account as the IAM role that the API uses (Kids First Open Data). This account happens to be different from the role where the API actually resides (Kids First STRIDES). This is necessary as compute used to be contained within Kids First Open Data but was migrated to the STRIDES account. However, the data did not move and the codebase required that the IAM used was owned by the same account as the data.
+
+#### Configuration
+
+Each study bucket needs to be configured with the correct security policies, encryption, replication, logging, and so on. For this reason, the bucket creation process was automated and included as part of the Data Tracker's service offerings. Upon creation, the Data Tracker will configure these additional settings ensuring all buckets are setup correctly.
+
+#### Layout
+Each study bucket has a similar layout:
+```
+source/
+  uploads/
+harmonized/
+```
+
+Under `source` are data files such as raw read files received from the sequencing center, MRI DICOMs from the investigator, and so forth. The `uploads` directory contains files that are uploaded from the Data Tracker. `harmonized` contains files that were processed by the BIX team using things such as a standard alignment workflow or calling pipeline.
+
+### Issues with Study Buckets
+
+## Storage Backend Solution
+By allowing users to create "Storage Backends" that grant Dewrangle access to bucket resources in other accounts, we will be able to provide more flexibility in how organizations choose to store their data.
+
+## Default Storage
+![enter image description here](https://i.ibb.co/vBb1YsN/storage-layout.png)
+"Default Storage" will refer to the defaults for how a new organization and its studies will be stored. It's important that new organizations and studies be quickly bootstraped into a working state without unnecessary configuration. To achieve this, Dewrangle will provide a default workspace for documents to land. This will be achieved by providing a Storage Backend that will point to Dewrangle's application bucket.
```

Comment:
```suggestion
![enter image description here](https://i.ibb.co/vBb1YsN/storage-layout.png)

"Default Storage" will refer to the defaults for how a new organization and its studies will be stored. It's important
that new organizations and studies be quickly bootstrapped into a working state without unnecessary
configuration. To achieve this, Dewrangle will provide a default workspace for documents to land. This will be
achieved by providing a Storage Backend that will point to Dewrangle's application bucket.
```
the formatting got weird